bin/migrate_tables.py

```diff
@@ -172,6 +193,27 @@ def validate_dest_args(
     return valid_source_args
 
 
+def write_logs(successful_migrations: List[TableRef], failed_migrations: List[TableRef]):
+    """Writes log files about script run.
+
+    Two logs files are written, one for successful migrations, one for unsuccessful. The logs are
+    formatted with each line having the destination [host].[database].[schema].[table]
+    """
+    timestamp = datetime.now().strftime("%Y%b%d_%H%M%S_%f").lower()
+    success_out_file = LOGS_DIR / f"migrate_tables-successful-{timestamp}.txt"
```

Comment:
Heads up that the `logs` folder needs to be committed/created if it doesn't exist during the run, cli runs fail otherwise.

Reply:
Committed empty `logs` dir